recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "17.0.0.dev0" %}
+{% set version = "17.0.0.rc1" %}
 # check https://clang.llvm.org/docs/OpenMPSupport.html
 # occasionally to see last fully supported openmp ver.
 {% set openmp_ver = "4.5" %}
```

Comment:
Before 17.0.0 GA we still need to decide what to do with the OpenMP version. LLVM 17 now defaults to 5.1, even though most of 5.0 & 5.1 are ostensibly not implemented yet. I asked about this upstream several times, leading to https://github.com/llvm/llvm-project/issues/63633, but no response so far.

See also #96 for some more details.